x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/components/group_manager_modal.tsx

```diff
@@ -36,7 +36,7 @@ import { GroupsLogic } from '../groups_logic';
 const ADD_SOURCE_BUTTON_TEXT = i18n.translate(
   'xpack.enterpriseSearch.workplaceSearch.groups.groupManagerUpdateAddSourceButton',
   {
-    defaultMessage: 'Add a Shared Source',
+    defaultMessage: 'Add an Organizational Source',
```

Comment:
Should this be Kibana-cased (only first word caps)
```suggestion
    defaultMessage: 'Add an organizational source',
```